tests/conftest.py

```diff
@@ -22,18 +22,152 @@
 from reana_workflow_controller.factory import create_app
 
 
-@pytest.fixture(scope='module')
+@pytest.fixture(scope="module")
+def tmp_shared_volume_path(tmpdir_factory):
```

Comment:
If you rebase all this fixtures should go away.

Reply:
Removed, opened a PR in pytest-reana to refactor the fixtures